src/main/java/nl/zandervdm/stayput/Repositories/PositionRepository.java

```diff
@@ -62,12 +71,148 @@ public Location getPreviousLocation(Player player, World world){
             return null;
         }
         if(position == null) return null;
-        double coordX = position.getCoordinate_x();
-        double coordY = position.getCoordinate_y();
-        double coordZ = position.getCoordinate_z();
+        double coordinate_x = position.getCoordinate_x();
```

Comment:
Please use the camelCase format for variables. This is meant for all the variables in your changes. 

(And if you find any of mine, please change them to if you want to :))